Libraries/ReactNative/PaperUIManager.js

```diff
@@ -33,6 +33,7 @@ function getConstants(): Object {
 function getViewManagerConfig(viewManagerName: string): any {
   if (
     viewManagerConfigs[viewManagerName] === undefined &&
+    global.nativeCallSyncHook && // If we're in the Chrome Debugger, let's not even try calling the sync method
```

Comment:
you can replace this with `DebugEnvironment`:

https://github.com/facebook/react-native/blob/1465c8f3874cdee8c325ab4a4916fda0b3e43bdb/Libraries/Utilities/DebugEnvironment.js